crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/CrimeApplyMapperTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.CrimeApplication;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.MaatApplication;
+import uk.gov.justice.laa.crime.applications.adaptor.testutils.FileUtils;
+import uk.gov.justice.laa.crime.applications.adaptor.testutils.JsonUtils;
+
+class CrimeApplyMapperTest {
+
+    private CrimeApplyMapper crimeApplyMapper;
+
+    @BeforeEach
+    void setUp() {
+        crimeApplyMapper = new CrimeApplyMapper();
+    }
+
+    @Test
```

Comment:
Is it possible to add few more test cases for the mapper to cover the scenarios such as when some of the application details are not present - e.g. provider details not present? Currently the test coverage for the mapper class is 79.1%